modules/auxiliary/admin/http/cnpilot_r_cmd_exec.rb

```diff
@@ -0,0 +1,142 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
```

Comment:
If you can execute arbitrary commands, this should definitely be an exploit module IMO.

Reply:
So, there might be an opportunity for a cmd shell version of this module, but iirc, cmd exec style modules (where you don't end up with a session) belong over here in Aux-land.

Reply:
AFAIU, one shot command runs, without any 'payloads'/'shell' stuff, would fall under aux...